Userland/Libraries/LibWeb/SVG/SVGViewPort.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2024, MacDue <macdue@dueutil.tech>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibWeb/SVG/AttributeParser.h>
+#include <LibWeb/SVG/ViewBox.h>
+
+namespace Web::SVG {
+
+class SVGViewPort {
```

Comment:
"viewport" is one word, so `SVGViewport` seems like the appropriate name here :)